### PR TITLE
Added InfoCard component

### PR DESCRIPTION
**Pull request by Hudayberdiev2000**

### CODE REVIEW

**Review by bruno-keiko — CHANGES_REQUESTED** [review]
Could you add a background color to the InfoCard story?

File: src/components/atoms/infoCard/InfoCard.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    ),
+    title: "7.30 AM - 9.30 PM",
+  },
+};
+
+const infoCard = {
+  title: "Atoms/InfoCard",
+  component: InfoCard,
+  tags: ["autodocs"],
+  excludeStories: /.*Data$/,
```
Comment (bruno-keiko) [inline src/components/atoms/infoCard/InfoCard.stories.tsx:28]:
```
  decorators: [
    (Story: FC) => (
      <div style={{ backgroundColor: " #c33" }}>
        <Story />
      </div>
    ),
  ],
  ```
You can use these lines to add background color.

File: src/components/atoms/infoCard/InfoCard.module.scss [hunk 2]
```diff
@@ -0,0 +1,23 @@
+@import "../../../foundations/sass/all";
+
+.info-card {
+  @include center;
+  gap: map-get($spacing, sm);
+  // background-color: $primary-color;
+  color: $white-floral;
+
```
Comment (bruno-keiko) [inline src/components/atoms/infoCard/InfoCard.module.scss]: What are your thoughts on adding the `display: flex` property? Could you review the design? I think the info card doesn't take up the full width of the screen.

File: src/components/atoms/infoCard/InfoCard.module.scss [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  &__icon {
+    width: map-get($spacing, mds);
+    height: map-get($spacing, mds);
+
+    svg {
+      width: 100%;
+      height: 100%;
+    }
+  }
+
```
Comment (bruno-keiko) [inline src/components/atoms/infoCard/InfoCard.module.scss:9]: Why do we need to set SVG width and height 100%?

File: src/foundations/sass/_variables.scss
Comment (bruno-keiko) [inline src/foundations/sass/_variables.scss:1]: Should we consider using spacing for margin, padding, and gap? And we can use `rem` directly with width and height.

**Comment by Hudayberdiev2000** [comment]
> Could you add a background color to the InfoCard story?

of course.

**Review by bruno-keiko — CHANGES_REQUESTED** [review]
Please avoid using color variables directly. Instead, create separate variables in the `_colors.scss` file for each component. This will help us to change colors easily for each component in the future.

**Comment by Hudayberdiev2000** [comment]
> Please avoid using color variables directly. Instead, create separate variables in the `_colors.scss` file for each component. This will help us to change colors easily for each component in the future.

got it

File: src/components/atoms/infoCard/InfoCard.module.scss (hunk 3 above)
Comment (Hudayberdiev2000) [reply src/components/atoms/infoCard/InfoCard.module.scss:9]: > Why do we need to set SVG width and height 100%?

i fixed the that in my own local git,

File: src/foundations/sass/_variables.scss
Comment (Hudayberdiev2000) [reply src/foundations/sass/_variables.scss:1]: > Should we consider using spacing for margin, padding, and gap? And we can use `rem` directly with width and height.

ok i'll use rem from now on.

File: src/components/atoms/infoCard/InfoCard.module.scss (hunk 2 above)
Comment (Hudayberdiev2000) [reply src/components/atoms/infoCard/InfoCard.module.scss]: I used flex just center the element, why it takes the full width, is there a problem ?

File: src/components/atoms/infoCard/InfoCard.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+          d="M10 0C4.5 0 0 4.5 0 10C0 15.5 4.5 20 10 20C15.5 20 20 15.5 20 10C20 4.5 15.5 0 10 0ZM14.2 14.2L9 11V5H10.5V10.2L15 12.9L14.2 14.2Z"
+          fill="#FFF8EE"
+        />
+      </svg>
+    ),
+    title: "7.30 AM - 9.30 PM",
+  },
+};
+
+const infoCard = {
```
Comment (bruno-keiko) [inline src/components/atoms/infoCard/InfoCard.stories.tsx]: infoCardConfig for clean code

**Review by bruno-keiko — APPROVED** [review]

**Pull request merged**